siakg/main_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/modules/tester"
+)
+
+// The tests in this file need to be checked manually, even if errors are
+// returned it will not be apparent to the test suite.
+//
+// CONTRIBUTE: Make these tests automatic.
```

Comment:
I thought about that. I'll probably do it. I might also add a global error variable, though that's pretty ugly.